LibreNMS/OS/Shared/Cisco.php

```diff
@@ -475,6 +473,20 @@ public function pollSlas($slas)
                     $tags = compact('rrd_name', 'rrd_def', 'sla_nr', 'rtt_type');
                     data_update($device, 'sla', $tags, $jitter);
                     $fields = array_merge($fields, $jitter);
+                    // Additional rrd for jitter packet loss percent
+                    $lossPercent = [
+                        'NumPackets' => $data[$sla_nr]['rttMonEchoAdminNumPackets'],
+                        'PacketLossSD' => $data[$sla_nr]['rttMonLatestJitterOperPacketLossSD'],
+                        'PacketLossDS' => $data[$sla_nr]['rttMonLatestJitterOperPacketLossDS'],
+                    ];
+                    $rrd_name = ['sla', $sla_nr, 'loss-percent'];
+                    $rrd_def = RrdDefinition::make()
+                        ->addDataset('NumPackets', 'GAUGE', 0)
+                        ->addDataset('PacketLossSD', 'GAUGE', 0)
+                        ->addDataset('PacketLossDS', 'GAUGE', 0);
+                    $tags = compact('rrd_name', 'rrd_def', 'sla_nr', 'rtt_type');
+                    data_update($device, 'sla', $tags, $lossPercent);
```

Comment:
need to change the data name or the tags otherwise this will have the same file name as the previous rrd file.

Reply:
I changed to $rrd_name on line 482 `$rrd_name = ['sla', $sla_nr, 'loss-percent'];` This appeared to pass through to the file name.